src/Page.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Button from './Button';
+import TodoList from './TodoList';
+import Input from './Input';
+import Title from './Title';
+import Container from './Container';
+
+const Page = ({
+  todos,
+  value,
+  onChangeValue,
+  addTodo,
+  deleteTodo,
+  inputRef,
+  checkTodo,
+}) => (
```

Comment:
```suggestion
const Page = ({
  todos, value, inputRef,
  onChangeValue, addTodo, deleteTodo,  checkTodo,
}) => (
```

`props` 가 많은 경우 줄바꿈을 해주면 좋은데 이정도로 비슷한 것 끼리 묶어서 보여줘도 괜찮을 것 같아요.

Reply:
이런 부분을 어떻게 정리하는지 매우 궁금했었는데 좋은 피드백 감사합니다 :) 변경했습니다!